templates/docs/examples/patterns/cta-section/default-25-75-links-within-text.html

```diff
@@ -0,0 +1,17 @@
+{% extends "_layouts/examples.html" %}
+{% from "_macros/vf_cta-section.jinja" import vf_cta_section %}
+{% block title %}CTA section / Full Width / Multiple Partial Links{% endblock %}
```

Comment:
```suggestion
{% block title %}CTA section / 25-75 / Multiple Partial Links{% endblock %}
```
I think this title needs to be adjusted?